Automattic/newspack-newsletters
# fix: change MC API key option name

### All Submissions:






### Changes proposed in this Pull Request:



https://github.com/Automattic/newspack-plugin/pull/1168 introduced a new name for the MC API key
option, which will be managed from the `newspack-plugin` instead of here. We do keep the old name for
backwards compatibility.

### How to test the changes in this Pull Request:

1. Set your MC API Key from `Newsletters > Settings > Mailchimp API Key`
2. Create a new newsletter.
3. Connect to your Mailchimp account, and see that the newsletter created is there as a draft campaign, which means that the API is working.
4. Pull this PR, and repeat 2. and 3. to create a second newsletter. Observe the newly created campaign is in your Mailchimp account as a draft campaign, which means the backward compatibility is 👌 
5. Create a new API Key from your Mailchimp account and set it in `Newsletters > Settings > Mailchimp API Key`
6. Create a third newsletter by repeating 2. and 3.
7. Observe that the third newsletter is in your Mailchimp account as a draft campaign, which means that the new API Key name is working.

### Other information:
